Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py

```diff
@@ -135,6 +136,26 @@ def get_direct_reports(self, user):
         res.pop('@odata.context', None)
         return res.get('value', [])
 
+    def get_manager(self, user):
+        manager_data = self.ms_client.http_request(
+            method='GET',
+            url_suffix=f'users/{user}/manager')
+        manager_data.pop('@odata.context', None)
+        return manager_data
+
+    #  If successful, this method returns 204 No Content response code.
+    #  Using resp_type=text to avoid parsing error.
+    def assign_manager(self, user, manager):
+        url = self.base_url
```

Comment:
don't think it's needed, you can remove

Reply:
See previous comment

Reply:
Updated, removed self.base_url = base_url by referring to the existing variable